src/System.EventLog/src/System/Diagnostics/SafeEventLogReadHandle.cs

```diff
@@ -0,0 +1,31 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.Win32.SafeHandles
+{
+    internal sealed class SafeEventLogReadHandle : SafeHandleZeroOrMinusOneIsInvalid
+    {
+        internal SafeEventLogReadHandle() : base(true) { }
+
+#pragma warning disable BCL0015�// Disable Pinvoke analyzer errors.
+        [DllImport(Interop.Libraries.Advapi32, CharSet = System.Runtime.InteropServices.CharSet.Unicode, SetLastError = true)]
+        internal static extern SafeEventLogReadHandle OpenEventLog(string UNCServerName, string sourceName);
+#pragma warning restore BCL0015
+
+#pragma warning disable BCL0015�// Disable Pinvoke analyzer errors. 
+        [DllImport(Interop.Libraries.Advapi32, SetLastError = true)]
+        private static extern bool CloseEventLog(IntPtr hEventLog);
+#pragma warning restore BCL0015
+
+        override protected bool ReleaseHandle()
```

Comment:
nit, should be `protected override` not `override protected`. see

https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md  bullet 5

Reply:
search elsewhere for other cases of `override protected` please

Reply:
Replaced everywhere